environment.yml

```diff
@@ -21,7 +21,8 @@ dependencies:
   - owslib
   - pandas
   - pip
-  - proj<8.0.0
+  - proj
```

Comment:
Good call. This was causing problems a few months ago but was dealt with recently.